include/itkPhaseAnalysisImageFilter.hxx

```diff
@@ -61,7 +61,22 @@ PhaseAnalysisImageFilter< TInputImage, TOutputImage >
 template< typename TInputImage, typename TOutputImage >
 void
 PhaseAnalysisImageFilter< TInputImage, TOutputImage >
-::DynamicThreadedGenerateData( const OutputImageRegionType & outputRegionForThread )
+::GenerateData()
```

Comment:
You don't need to do this. This just duplicates the code in `ImageSource::GenerateData()`

Reply:
`DynamicThreadedGenerateData` will still be indirectly called by your derived class.